indicators/santander-951d27d.yml

```diff
@@ -0,0 +1,27 @@
+title: Santander Phishing Kit 951d27d
+description: |
+    Detects a Santander phishing kit targeting Spanish speaking users.
+    
+references:
+    - https://urlscan.io/result/d7f3f389-d10b-4b83-a45c-ba7f8ec54035
+    - https://urlscan.io/result/1c849740-38f2-4442-94f8-bf2147cc587e
+    
+detection:
+
+    cloneTimestamp:
+      requests|contains: '?v=1655293257536'
+      
+    usernameLabelID:
+      html|contains: '47563B2825160654ADB2CC97CE152AF3'
```

Comment:
This is a good example of where #38 could be problematic. For these sites the [HTML](https://urlscan.io/responses/010f8c68c2cf79fa6fa3a7b31f014555b474e375c58c154fbdc62dd3c4c1cdef/) and [DOM](https://urlscan.io/dom/d7f3f389-d10b-4b83-a45c-ba7f8ec54035/) are very different

The HTML is the static stuff loaded from the server whereas it seems this kit builds most of the page dynamically using JS.

Let's add this rule for now, but I'll keep a note to update it once we've got DOM support

Reply:
I'm not sure this kit is dynamically generated by JS as you have suggested, I see nothing to indicate such behaviour. Also managed to see why the 2nd reference did not flag, it because the username & password label IDs are not present in the DOM, will adjust the rule accordingly now.